### DISCUSSION TITLE
Support more social link icons

### DISCUSSION THREAD

**Discussion by lomirus** [CaiJimmy/hugo-theme-stack#774] [Ideas]
(It seems that the feature request is not allowed in Issues now? There is only a bug feedback option)

Hello, @CaiJimmy. I just saw your blog and noticed there is a link of mastodon icon. However, this blog theme have not supported mastodon social link yet. It only supports twitter and github. And I guess  it would not be so difficult to add the mastodon icon since you have had the icon, right? And besides the mastodon, some other social links like Discord, Matrix, Reddit and Email are worth considering. Thanks.

**Comment by lomirus** [comment]
Well, I just found this discussion #630 said no new features will be add anymore. Sorry for the bothering 😶.

**Comment by CaiJimmy** [comment]
https://stack.jimmycai.com/config/menu#add-custom-icon

You can add more icons yourself.

**Reply by lomirus** [reply]
Thanks!
